src/main/java/com/ericsson/ec/subscriptionobject/SubscriptionObject.java

```diff
@@ -0,0 +1,87 @@
+package com.ericsson.ec.subscriptionobject;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
Although Jackson is used in EI so is org.json, maybe we could keep it to just one JSON-handler here, org.json is easier to build and change as I understand. Jackson best advantage in EI is that is was included in Spring.

Reply:
Fixed, i thought i had problems with the normal JSON library but i worked fine now.